.env

```diff
@@ -1 +1 @@
-ESLINT_NO_DEV_ERRORS=true
+VITE_API_KEY=AIzaSyD0ftUETmycY3ljtwDjD7cyvdAeEKENdI0
```

Comment:
env shouldn't be on GH
it's critical error
If this was a test for a job, you would have been rejected.

never add sensitive information to Gh there are other ways to do that